sigma/validators/sigmahq/tags.py

```diff
@@ -91,3 +94,47 @@ def validate(self, rule: SigmaRuleBase) -> List[SigmaValidationIssue]:
             if tag.namespace == "tlp" and tag.name not in self.allowed_tlp:
                 return [SigmahqTagsTlpIssue([rule], tlp=tag.name)]
         return []
+
+
+@dataclass
+class SigmahqTagsTechniquesWithoutTacticsIssue(SigmaValidationIssue):
+    description: ClassVar[str] = (
+        "A MITRE ATT&CK technique tag was found without its corresponding tactic tag."
```

Comment:
```suggestion
        "A MITRE ATT&CK technique tag was found without its corresponding tactic name tag."
```
https://attack.mitre.org/tactics/TA0003/ so can be attack.ta0003 for tag instead of "persistence".